src/test/java/cz/cvut/kbss/analysis/dao/FaultTreeDaoTest.java

```diff
@@ -116,26 +116,26 @@ public void symmetricBranches_shouldQueryAll() {
         transactional(() -> faultTreeDao.update(tree)); // probabilities propagation update (simulation)
 
         Optional<FaultTree> optionalTree = faultTreeDao.find(tree.getUri());
-        if(!optionalTree.isPresent()) Assert.fail("Tree was not queried");
+        if(optionalTree.isEmpty()) Assertions.fail("Tree was not queried");
 
         FaultTree queriedTree = optionalTree.get();
         FaultEvent manifestingEvent = queriedTree.getManifestingEvent();
-        Assert.assertNotNull(manifestingEvent);
+        Assertions.assertNotNull(manifestingEvent);
 
         Set<FaultEvent> rootChildren = manifestingEvent.getChildren();
-        Assert.assertFalse(rootChildren.isEmpty());
+        Assertions.assertFalse(rootChildren.isEmpty());
 
         rootChildren.forEach(child -> {
-            Assert.assertEquals(1, child.getChildren().size());
+            Assertions.assertEquals(1, child.getChildren().size());
             FaultEvent childX = child.getChildren().iterator().next();
-            Assert.assertEquals("X", childX.getName());
+            Assertions.assertEquals("X", childX.getName());
 
             Set<FaultEvent> xChildren = childX.getChildren();
-            Assert.assertFalse(xChildren.isEmpty());
```

Comment:
Ok although I like to have static import:
`import static org.junit.jupiter.api.Assertions.*;`

So you would not have to have Assert.assertFalse() but just assertFalse()

Reply:
Done